packages/components/filters/filter-menu/src/filter-menu.readme.mdx

```diff
@@ -0,0 +1,6 @@
+import { Meta, Markdown } from '@storybook/blocks';
+import ReadMe from './../README.md?raw'
+
+<Meta tags={['local-dev']} title="components/Filters/FilterMenu/Readme" />
```

Comment:
@misama-ct we can use `tags` in the functional-component flavor of `Meta` for `*.mdx`.  I think I prefer this over a separate glob-based approach in `main.ts`.

Reply:
Very nice. Perfect.